examples/simple/posix/UB-PTHREAD6-bad.output

```diff
@@ -0,0 +1,7 @@
+Error: UB-PTHREAD6
+Description: Attempting to unlock a PTHREAD_MUTEX_DEFAULT or PTHREAD_MUTEX_NORMAL mutex which is not robust and which has not been locked by the current thread.
```

Comment:
OK then.  What I didn't know was that `robust` is a technical term in this context.  Hopefully others will not get confused.